From c0db8428968709ebab1d7408a5d4e22e3b0ee7e7 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Tue, 26 Nov 2024 10:58:22 +0000
Subject: [PATCH] Revert "Make device as unlocked state by default on userdebug
 build for civ"

This reverts commit 0f7383ffe04d70b938893a06aa6f5295eab78acd.

Tracked-On: NA
Signed-off-by: NA
---
 kernelflinger.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernelflinger.c b/kernelflinger.c
index 5378384..fd70a6d 100644
--- a/kernelflinger.c
+++ b/kernelflinger.c
@@ -1528,7 +1528,9 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table)
 #ifndef USER
 	/* WA patch to set device as unlocked by default for userdebug build
 	 */
+#ifdef USE_SBL
 	set_current_state(UNLOCKED);
+#endif
 #else
 	/* For civ, flash images to disk is not MUST. So set device to LOCKED
 	 * state by default on the first boot.
-- 
2.34.1

